75521 rustdoc book improvements (#75778)
rust-lang/rust: src/doc/rustdoc/src/what-is-rustdoc.md

## Basic usage

Let us give it a try! Create a new project with Cargo:

```bash
$ cargo new docs
$ cd docs
```

In `src/lib.rs`, Cargo has generated some sample code. Delete
it and replace it with this:

```rust
/// foo is a function
fn foo() {}
```

> Now, run `rustdoc` on our code. To do so, we can call it with the path to
> our crate root like this:

Reviewer: This makes it inconsistent: 'run rustdoc on our code' sounds odd without "let's". I still prefer "let's", but if you want to keep it this way you could say something like
```suggestion
Now you can run `rustdoc` on your code. To do so, call it with the path to
the crate root like this:
```

Author: I think I'm going to return a measure of contractions.
